release-notes/opensearch-dashboards.release-notes-2.2.0.md

```diff
@@ -0,0 +1,46 @@
+## Version 2.2.0 Release Notes
+
+#### Notable changes
+* Bump node version to 14.20.0 ([#2101](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2101))
+* OpenSearch Dashboards uses [OUI](https://github.com/opensearch-project/oui) and its alias onto EUI ([#2080](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2080))
+* New experimental feature: adds the Drag and Drop editor to Visualize. Note this is disabled by default. Please enable by setting `wizard.enabled: true` in `opensearch_dashboards.yml` ([#1966](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1966))
+
+#### Deprecations
+* Deprecate the Blacklist / Whitelist nomenclature ([#1808](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1808))
+
+### 📈 Features/Enhancements
+* Add DocView links pluggable injection capability ([#1200](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1200))
+* Enable users to select custom vector map for visualization ([#1718](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1718))
+* [UX] Restyle global breadcrumbs ([#1954](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1954))
+* [Feature] Adds the Drag and Drop editor to Visualize ([#1966](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1966))
```

Comment:
Can we make all bullet point items one tense and conjugation? So, "Add DocView...", "Add the Drag&Drop", "Update Code of Conduct". Or "Adds DocView", "Adds the Drag&Drop", "Updates Code of Conduct"?